pkg/jobs/utils/helpers.go

```diff
@@ -559,3 +561,48 @@ func GetMonitorAttempts(jobType string, curator *clustercuratorv1.ClusterCurator
 
 	return monitorAttempts
 }
+
+// Copied from https://github.com/stolostron/library-go which is no longer supported
+// but this is a useful func for local testing as we don't need to build the image
+// LoadConfig loads the kubeconfig and returns a *rest.Config
+// url: The url of the server
+// kubeconfig: The path of the kubeconfig, if empty the KUBECONFIG environment variable will be used.
+// context: The context to use to search the *rest.Config in the kubeconfig file,
+// if empty the current-context will be used.
+// Search in the following order:
+// provided kubeconfig path, KUBECONFIG environment variable, in the cluster, in the user home directory.
+// If the context is not provided and the url is not provided, it returns a *rest.Config the for the current-context.
+// If the context is not provided but the url provided, it returns a *rest.Config for the server identified by the url.
+// If the context is provided, it returns a *rest.Config for the provided context.
+func LoadConfig(
+	url,
+	context string,
```

Comment:
We don't need the context or url params. They are always empty in our usage right?